setup.py

```diff
@@ -7,7 +7,7 @@
 
 DEPENDENCIES = [
     'applicationinsights',
-    'azure-cli',
+    'azure-cli==2.0.2',
```

Comment:
Shouldn't pin to a specific CLI version as this doesn't work as intended.
You'll get all the latest modules still.